package.json

```diff
@@ -70,6 +72,7 @@
     "cozy-realtime": "^5.6.2",
     "date-fns": "2.29.3",
     "events": "^3.3.0",
+    "flexsearch": "^0.7.43",
```

Comment:
I don't think flexsearch and comlink are necessary here? There are both dependencies from cozy-dataproxy-lib

Reply:
Nice catch, I don't remember why I added them, fixed [here](https://github.com/cozy/cozy-flagship-app/compare/219369d4369a17e3edea52d51acac05ebd6f8687..235beab340692dc5a5c5930440ed84489bf8fe55)